saleor/plugins/base_plugin.py

```diff
@@ -228,6 +234,24 @@ def __str__(self):
 
     capture_payment: Callable[["PaymentData", Any], GatewayResponse]
 
+    #  Trigger when category is created.
+    #
+    #  Overwrite this method if you need to trigger specific logic after a category is
+    #  created.
+    category_created: Callable[["Category", Any], Any]
```

Comment:
Instead of `Any`, can we use a real `previous_value` and response here? For both of them, this will be `None` probably
(I know that the rest of calls have `Any` everywhere, but at least for the new one, we could provide better declaration

Reply:
done